stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt

```diff
@@ -189,3 +192,8 @@ internal fun DefaultMessagesLoadingMoreIndicator() {
             .padding(8.dp)
     )
 }
+
+/**
+ * Default value for [Messages] content padding.
+ */
+internal val DefaultMessagesContentPadding: Dp = 16.dp
```

Comment:
As mentioned above, I'd push this to `StreamDimens` and then fetch it from the chat theme rather than having a constant in the code